crates/broker/src/lib.rs

```diff
@@ -825,7 +825,7 @@ where
             tracing::info!("Configured to run with Bento backend");
 
             Arc::new(
-                provers::Bonsai::new(config.clone(), bento_api_url.as_ref(), "")
+                provers::Bonsai::new(config.clone(), bento_api_url.as_ref(), "reserved:999")
```

Comment:
This can starve out all the proof requests made by the boundless-cli (reserved=1) unless people know to manually set the it when making a boundless-cli call. I can see a bunch of people complaining about why their povw prepares and submits are failing or never completing lol

Reply:
It just means it will only use one prove agent at a time, to avoid starving the prove agents from completing orders. I could probably increase the reserved value a bit so that it doesn't cap out at 1, and so that when capped it doesn't conflict with mining, but I don't see what issue you have with setting this high of a reservation. This is needed to ensure any non-reserved work does not use resources for orders.

Reply:
Have the same concern as Eric. I don't know the details of how the reserved mechanism works, but it sounds like you are saying that even if there is a backlog of proofs with '999 reserved', a prepare/submit proof with 1 reserved will get weaved in in a reasonable time frame - if this is the case then sounds good!

Reply:
That is correct. 1 reserved would be prioritized if at least 1 GPU prove agent is being used for the other. Because avoiding taskdb changes ATM, this leads to a bit unideal situation with only 1 GPU, because it would just be ordered by segment creation time. I changed this value to 5, so this would by default use up to 5 gpus, and I'm biasing to lower amount to not disrupt the broker capacity assumption, but I think I might increase it more such that it doesn't get delayed by mining too much